tests/testthat/test-add_gloabl_p.tbl_uvregression.R

```diff
@@ -0,0 +1,52 @@
+skip_on_cran()
+test_that("no errors/warnings with standard use after tbl_uvregression", {
+  tbl <- trial %>% tbl_uvregression(method = lm, y = marker, include = c("age", "grade"))
+
+  expect_silent(
+    res <- tbl %>% add_global_p()
+  )
+
+  expect_snapshot(res %>% as.data.frame())
+
+  # two model terms, two p values
+  expect_equal(
```

Comment:
Can you also add a test that the p-values match those returned from `car::Anova()`?